run_kraken.py

```diff
@@ -49,6 +49,8 @@ def inject_node_scenario(action, node_scenario, node_scenario_object):
     node_name = node_scenario.get("node_name", "")
     label_selector = node_scenario.get("label_selector", "")
     timeout = node_scenario.get("timeout", 120)
+    service = node_scenario.get("service", "")
+    ssh_private_key = node_scenario.get("ssh_private_key", "")
```

Comment:
Maybe lets add "~/.ssh/id_rsa" as the default value for the ssh_private_key parameter

Reply:
Sure, added it.